docs/gallery_scripts_template/plot_peakmap_binning_demonstration_ms_matplotlib.py

```diff
@@ -0,0 +1,43 @@
+"""
+Plot Peakmap Binning Demonstration
+==================================
+
+This example demonstrates how different binning levels affect peak map visualization
+"""
+
+import pandas as pd
+import requests
+from io import StringIO
+import numpy as np
+import matplotlib.pyplot as plt
+
+pd.options.plotting.backend = "ms_matplotlib"
+
+url = "https://github.com/OpenMS/pyopenms_viz/releases/download/v0.1.5/TestMSExperimentDf.tsv"
+response = requests.get(url)
+response.raise_for_status()
+df = pd.read_csv(StringIO(response.text), sep="\t")
+
+
+fig, axs = plt.subplots(1, 3, figsize=(18, 6), sharex=True, sharey=True)
+
+
+binning_levels = [(10, 10), (50, 50), (100, 100)]
+
+for ax, (num_x_bins, num_y_bins) in zip(axs, binning_levels):
+    
+    df.plot(
```

Comment:
Please change to match https://github.com/OpenMS/pyopenms_viz/commit/898b635c8142241f6e6ef62661c2ea597263fd8f
so plotting works on scripts as well. 

This means changing show_plot=False and adding plt.show() to the end